diff --git a/jellyfin_kodi/jellyfin/http.py b/jellyfin_kodi/jellyfin/http.py
--- a/jellyfin_kodi/jellyfin/http.py
+++ b/jellyfin_kodi/jellyfin/http.py
@@ -202,7 +202,10 @@
 
         self._get_header(data)
         data["timeout"] = data.get("timeout") or self.config.data["http.timeout"]
-        data["verify"] = data.get("verify") or self.config.data.get("auth.ssl", False)
+        if data.get("verify") is None:
+            data["verify"] = self.config.data.get("auth.ssl")
+        if data["verify"] is None:
+            data["verify"] = True
         data["url"] = self._replace_user_info(data["url"])
         self._process_params(data.get("params") or {})
         self._process_params(data.get("json") or {})
